x-pack/filebeat/input/netflow/testdata/golden/IPFIX-Barracuda-extended-uniflow-template-256.golden.json

```diff
@@ -0,0 +1,153 @@
+{
+  "test_name": "IPFIX Barracuda extended uniflow template 256",
+  "events": [
+    {
+      "Timestamp": "2018-04-18T08:16:47Z",
+      "Meta": null,
+      "Fields": {
+        "destination": {
+          "ip": "64.235.151.76",
+          "locality": "public",
+          "port": 443
+        },
+        "device": {
+          "ip": "192.0.2.1"
+        },
+        "event": {
+          "action": "netflow",
+          "created": "2018-04-18T08:16:47Z",
+          "duration": 0
+        },
+        "flow": {
```

Comment:
ECS doesn't define `flow` at this time. This is risky business, because it's a likely addition for the future :-)

I'd map the ID to `event.id` instead.

Reply:
Well, that is a problem.

Reply:
`event.id` isn't really a good fit for this type of value because it's not unique to this one event. It's an ID that's unique to the characteristics of the flow. Another flow that goes between `A <-> B` on the same ports, vlans, etc. will have the same value.